tools/ConvertTo-PseudoLocalization.ps1

```diff
@@ -60,19 +60,16 @@ $content.PreserveWhitespace = $true
 $content.Load($Path)
 
 function GetPseudoLocalization([string]$key, [string]$value, [string]$comment) {
-    $locked = $null
-    if ($comment -match '.*\{Locked=?([^}]*)\}.*') {
-        $locked = $Matches[1]
-    }
+    $placeholders = @{}
 
-    # Skip {Locked} and {Locked=qps-ploc} entries
-    if ($locked -and (($locked -eq '') -or $locked.Contains('qps-ploc'))) {
-        continue
-    }
+    if ($comment -match '.*\{Locked=?([^}]*)\}.*') {

```

Comment:
I figured running the `-match` operator right here in the `if` is way easier to read.